Rakefile

```diff
@@ -0,0 +1,12 @@
+begin
+  require 'rspec/core/rake_task'
+  RSpec::Core::RakeTask.new(:spec)
+
+  require 'rubocop/rake_task'
+  RuboCop::RakeTask.new(:rubocop)
+
+  task default: [:spec, :rubocop]
```

Comment:
Should the default be set outside of the `begin/require` block?  It might be strange that the default would be different under different contexts.